python/examples/core/1-first-steps.py

```diff
@@ -0,0 +1,264 @@
+"""
+.. _core-tutorial-first-steps:
+
+First steps with metatensor
+===========================
+
+This tutorial explores how data is stored inside metatensor's ``TensorMap``, and how to
```

Comment:
```suggestion
This tutorial explores how data is stored inside metatensor's `:py:class:`TensorMap`, and how to
```

Reply:
we can not do that here, otherwise it renders weird on the tutorials list.